### DISCUSSION TITLE
Return the Youtube comments on same place..

### DISCUSSION THREAD

**Discussion by BateauSD** [lightbeam24/CustomTube#190] [Ideas]
Hi. Around one week ago Youtube replaced comments with recommends videos.. and it looks really ugly.
Can you do something please about it to fix?
![image](https://github.com/lightbeam24/CustomTube/assets/13334848/b7f4cafa-e87f-4202-b84a-a0404b954708)

**Comment by lightbeam24** [comment]
1.3.0 Beta 2 will fix this. It's almost done and might release today or tomorrow.

**Comment by lightbeam24** [comment]
Do you still have this new layout? I no longer have it, and it seems like most others don't either. If they really have reverted it for everyone, then I don't want to rush Beta 2 out (currently, it's not really much of an update).

**Comment by Zhrak81** [comment]
They seem to be testing the new layout in different regions. I was met with it this morning and man does it mess up the various addons and scripts I have in place to "fix" YouTube.

**Comment by BateauSD** [comment]
Seems youtube fixed that by own. Logged today and comments loaded as usual.
I hope they will not do that again.
